CodeEdit/Features/WindowCommands/NavigateCommands.swift

```diff
@@ -12,7 +12,7 @@ struct NavigateCommands: Commands {
         CommandMenu("Navigate") {
             Group {
                 Button("Reveal in Project Navigator") {
-
+                    NSApp.sendAction(#selector(ProjectNavigatorViewController.revealFile(_:)), to: nil, from: nil)
```

Comment:
If you could enable this command by adding `.disabled` to the other buttons in NavigateCommands that'd be great. They're only disabled because there was no implementation.